scrapy/commands/genspider.py

```diff
@@ -66,6 +66,10 @@ def run(self, args, opts):
             print("Cannot create a spider with the same name as your project")
             return
 
+        if exists(name + ".py"):
+            print("Spider %r already exists in module" % name)
```

Comment:
This message is not entirely accurate, as any file with the same name would trigger it (not necessarily a spider).

Reply:
Do you think a message like "File %r.py already exists in the current directory" would be more accurate?

Reply:
I think talking about the file is indeed better, thanks for making the change. I don't think adding a `--force` option is necessary, in my opinion it'd just make the code more complex for something that could be solved easily outside of Scrapy (renaming the existing file or choosing a different name for the new file).

Reply:
I agree, so I guess no more changes here then?